### PR TITLE
Node state transition diagram fixes

### PR DESCRIPTION
**Pull request by kmdalal**
### Summary ###

- Updates inaccurate node state diagrams for Plexil4 (5 diagrams).
- Adds the Plexil 6 version of the Waiting diagram, and points out the difference with Plexil 4.
- Corrects diagram captions and removes unneeded ones.
- Removes the unneeded `alt` tag from diagram entries.

### CODE REVIEW

File: Appendices/NodeStateDiagrams.rst [hunk 1]
```diff
@@ -76,7 +69,6 @@ Assignment nodes transition to ITERATION_ENDED when the ExitCondition
 becomes true; the PostCondition determines the outcome.
 
 .. figure:: ../_static/images/Executing_Assignment_nodes_with_Exit_condition.png
```
Comment (plexilcrf) [inline Appendices/NodeStateDiagrams.rst]: Per our conversation, this link needs to be updated to the new diagram you created.

File: Appendices/NodeStateDiagrams.rst [hunk 2]
```diff
@@ -19,33 +15,31 @@ areas.
 INACTIVE state
 --------------
 
-Effective with the |PLEXIL| 3 release, INACTIVE nodes behave as shown:
+Nodes of all types in the INACTIVE nodes behave as follows.
```
Comment (plexilcrf) [inline Appendices/NodeStateDiagrams.rst]: Should be 'in the INACTIVE state'

File: Appendices/NodeStateDiagrams.rst (hunk 1 above)
Comment (kmdalal) [reply Appendices/NodeStateDiagrams.rst]: Done

File: Appendices/NodeStateDiagrams.rst (hunk 2 above)
Comment (kmdalal) [reply Appendices/NodeStateDiagrams.rst]: Done

**Review by plexilcrf — APPROVED** [review]

**Pull request merged**